src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/cloud/IridaFileStorageAwsUtilityTest.java

```diff
@@ -178,6 +178,22 @@ public void testWriteFile() throws IOException {
 		}
 	}
 
+	@Test
+	@Override
+	public void testDeleteFile() {
+		iridaFileStorageUtility.deleteFile(PATH_TO_FASTA_FILE);
+		boolean fileExistsInBlobStorage = iridaFileStorageUtility.fileExists(PATH_TO_FASTA_FILE);
+		assertFalse(fileExistsInBlobStorage, "File should not exist in azure blob storage");
```

Comment:
Update the text to `File should not exist in the aws s3 bucket` since this is the AWS test

Reply:
Please see 8ff3e0d0d232cee5282bf49e2e87b4d02b9c88d2.